modules/exploits/osx/local/feedback_assistant_root.rb

```diff
@@ -55,11 +55,16 @@ def upload_executable_file(filepath, filedata)
 
   def check
     version = Rex::Version.new(get_system_version)
+
+    return CheckCode::Unknown('Could not retrieve OSX version') if version.blank?
```

Comment:
I believe `Rex::Version.new(...).blank?` will always return false

```
>> Rex::Version.new(nil).blank?
=> false
>> Rex::Version.new("1.1.1").blank?
=> false
>> Rex::Version.new("0").blank?
=> false

```

Reply:
Oops. That's what I had the first time.